geocoding/location-service/run_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+ENV=`mktemp -d`
+python3 -mvenv "${ENV}"
+. "${ENV}/bin/activate"
+pip install -r requirements.txt
+ENABLE_FAKE_GEOCODER=YES pytest
+deactivate
```

Comment:
Not a blocker but I think it'd be better to use Poetry for better dependency and virtual environment management

Reply:
OK, I've changed it to poetry in running the tests, building the docker image, and CI.